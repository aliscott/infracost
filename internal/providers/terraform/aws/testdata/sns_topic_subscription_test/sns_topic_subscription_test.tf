provider "aws" {
  region                      = "us-east-1" # <<<<< Try changing this to eu-west-1 to compare the costs
  skip_credentials_validation = true
  skip_requesting_account_id  = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
}

resource "aws_sns_topic_subscription" "sns_topic_subscription" {
  endpoint = "some-dummy-endpoint"
  protocol = "http"
  topic_arn = "arn:aws:sns:us-east-1:123456789123:sns-topic-arn"
}

resource "aws_sns_topic_subscription" "sns_topic_subscription_withUsage" {
  endpoint = "some-dummy-endpoint"
  protocol = "http"
  topic_arn = "arn:aws:sns:us-east-1:123456789123:sns-topic-arn"
}