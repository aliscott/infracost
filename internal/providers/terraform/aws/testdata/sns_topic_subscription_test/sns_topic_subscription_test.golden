
 Name                                                                Monthly Qty  Unit                      Monthly Cost 
                                                                                                                         
 aws_sns_topic_subscription.sns_topic_subscription                                                                       
 └─ HTTP notifications                                        Monthly cost depends on usage: $0.60 per 1M notifications  
                                                                                                                         
 aws_sns_topic_subscription.sns_topic_subscription_withUsage                                                             
 └─ HTTP notifications                                                         2  1M notifications                 $1.20 
                                                                                                                         
 PROJECT TOTAL                                                                                                     $1.20 

----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file